src/sentry/auth/providers/saml2.py

```diff
@@ -29,6 +34,52 @@ def OneLogin_Saml2_Auth(*args, **kwargs):
     def OneLogin_Saml2_Settings(*args, **kwargs):
         raise NotImplementedError('Missing SAML libraries')
 
+    class OneLogin_Saml2_Constants(object):
+        def __getattr__(self):
+            return None
+
+
+IDENTIFIER_CHOICES = (
```

Comment:
Since we won't be supporting username identifiers this just becomes email. I'm going to remove this.